app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/controller/SearchController.java

```diff
@@ -24,6 +35,10 @@
     private Index index;
     private Gson gson;
     private static SearchController instance;
+    private final String GOOGLE_BOOK_URL = "https://www.googleapis.com/books/v1/volumes?&maxResults=1&projection=lite&q=";
+    private final String apiKey = "AIzaSyA0Km9mozkCqNX7N0Sy4V_Uk1OIvxUvRn4";
+    private Book isbnBook;
```

Comment:
I don't think isbnBook makes sense as instance var. If you switch to CompleteableFuture this won't be required